package.json

```diff
@@ -22,6 +22,6 @@
   },
   "homepage": "https://github.com/chadlieberman/graphql-type-long#readme",
   "dependencies": {
-    "graphql": "^0.9.1"
+    "graphql": "^0.9.1 || ^0.10.0"
```

Comment:
Alternatively this could be:
* `>=0.9.1`, but that would allow for `1.*.*` updates as well.
* `>=0.9.1 <1.0.0` but that would allow for `0.11.*` updates as well.

Happy to use any of these that you feel fit.  The current one here is the most restrictive of the three options, any of which would work for me.